src/Nethermind/Nethermind.Synchronization/ParallelSync/SyncProgressResolver.cs

```diff
@@ -36,6 +37,7 @@ public class SyncProgressResolver : ISyncProgressResolver
         // then we will never have to look 128 back again
         // note that we will be doing that every second or so
         private const int MaxLookupBack = 128;
+        public static readonly Keccak LatestBlockWithFullState = Keccak.Compute(Encoding.ASCII.GetBytes("LatestBlockWithFullStateKey"));
```

Comment:
No need to be _public_ ?
Add a comment.

Reply:
Should be `private`